Makefile

```diff
@@ -1,7 +1,7 @@
 NAME      := mpifileutils
 SRC_EXT   := gz
 REPO_NAME := mpifileutils-pkg
-PKG_GIT_COMMIT := 98017cf8c38ae99d47cb3d507d69e7cf87fa6f26
+PKG_GIT_COMMIT := 3975d8483c2471c59c599d87516c7cea24bcc032
```

Comment:
It's a real pity that it's been almost 2 years since a new release has been made by upstream.